yarn.lock

```diff
@@ -2951,6 +2951,7 @@ __metadata:
     "@cardano-sdk/crypto": ^0.1.4
     "@cardano-sdk/dapp-connector": ^0.9.0
     "@cardano-sdk/key-management": ^0.5.1
+    "@cardano-sdk/tx-construction": ^0.4.0
```

Comment:
Can we use the `"workspace:^"` notation?

Reply:
We should do that in https://github.com/input-output-hk/cardano-js-sdk/pull/720 after this is merged and that branch is rebased